libdnf/dnf-sack.cpp

```diff
@@ -772,13 +772,24 @@ load_yum_repo(DnfSack *sack, HyRepo hrepo, GError **error)
         fp_primary = solv_xfopen(primary.c_str(), "r");
         assert(fp_primary);
 
-        g_debug("fetching %s", name);
-        if (repo_add_repomdxml(repo, fp_repomd, 0) || \
-            repo_add_rpmmd(repo, fp_primary, 0, 0)) {
+        g_debug("%s: loading: %s", __func__, fn_repomd);
```

Comment:
If you wanted to remove the function names altogether, they are still here. No big deal, I'd also phrase the error messages (referring to all the messages in the last force-push in general) differently, but let's leave it the way it is, I'll be a bit more nitpicky in dnf 5 :slightly_smiling_face:

Reply:
okok, for now I have just removed the remaining added function names.